### PR TITLE
Bc module sensor

### PR DESCRIPTION
**Pull request by blavka**

### CODE REVIEW

File: bcl/src/bc_module_sensor.c [hunk 1]
```diff
@@ -179,3 +185,74 @@ void bc_module_sensor_toggle_output(bc_module_sensor_channel_t channel)
...
+            return BC_MODULE_SENSOR_REVISION_UNKNOWN;
+        }
+
+        uint8_t test_vector = 0x86;
+        int value_input;
+        int value_output;
+
+        for (size_t i = 0; i < sizeof(test_vector); i++)
+        {
+            value_output = ((test_vector >> i) & 0x01);
```
Comment (hubpav) [inline bcl/src/bc_module_sensor.c]: Extra brackets

File: bcl/src/bc_module_sensor.c [hunk 2]
```diff
@@ -179,3 +185,74 @@ void bc_module_sensor_toggle_output(bc_module_sensor_channel_t channel)
...
+
+            if (!bc_tca9534a_write_pin(&_bc_module_sensor.tca9534a, BC_TCA9534A_PIN_P0, value_output))
+            {
+                return BC_MODULE_SENSOR_REVISION_UNKNOWN;
+            }
+
```
Comment (hubpav) [inline bcl/src/bc_module_sensor.c]: Extra line

File: bcl/src/bc_module_sensor.c [hunk 3]
```diff
@@ -179,3 +185,74 @@ void bc_module_sensor_toggle_output(bc_module_sensor_channel_t channel)
...
+    if (port_actual != port_new)
+    {
+        return bc_tca9534a_write_port(&_bc_module_sensor.tca9534a, port_new);
+    }
+
+    return true;
+}
+
+bc_module_sensor_revision_t bc_module_sensor_get_revision(void)
```
Comment (hubpav) [inline bcl/src/bc_module_sensor.c]: How does it work, is it using parasite capacitance of the pin?

File: bcl/src/bc_module_sensor.c (hunk 2 above)
Comment (blavka) [reply bcl/src/bc_module_sensor.c]: I know

File: bcl/src/bc_module_sensor.c (hunk 3 above)
Comment (hubmartin) [reply bcl/src/bc_module_sensor.c]: two pins of the expander are connected with 10k resistor https://github.com/bigclownlabs/bc-hardware/blob/master/out/bc-module-sensor/bc-module-sensor-rev-1-1-sch.pdf

**Pull request merged**